src/wallet/wallet.cpp

```diff
@@ -3138,6 +3143,27 @@ int64_t CWallet::GetOldestKeyPoolTime()
     return oldestKey;
 }
 
+void CWallet::MarkDestinationTransactionsDirty(const CTxDestination target_dst)
+{
+    LOCK(cs_wallet);
+
+    for (auto& entry : mapWallet) {
+        CWalletTx& wtx = entry.second;
+
+        for (unsigned int i = 0; i < wtx.tx->vout.size(); i++)
+        {
```

Comment:
Nit: `{` on same line as `for`; `++i` over `i++`.

Reply:
done